src/marqo/tensor_search/validation.py

```diff
@@ -85,6 +90,29 @@ def validate_bulk_query_input(q: 'BulkSearchQueryEntity') -> Optional[MarqoError
 
     return None
 
+def validate_searchable_attributes(field_names: List[str], searchable_attributes: Optional[List[str]], search_method: SearchMethod):
```

Comment:
Adding a docstring would be good

Reply:
devs would need context/explanation for this, because there's logic which means it's not a super-general function (e.g.: this function assumes MARQO_MAX_SEARCHABLE_TENSOR_ATTRIBUTES is set)

Reply:
making the name more specific would also help (it's not a general searchable_attributes validation function)